client/model.go

```diff
@@ -40,6 +40,73 @@ type ProjectCreatePayload struct {
 	Description string `json:"description"`
 }
 
+type EnvironmentCreate struct {
+	Name                        string         `json:"name"`
+	ProjectId                   string         `json:"projectId"`
+	DeployRequest               *DeployRequest `json:"deployRequest"`
+	WorkspaceName               string         `json:"workspaceName,omitempty"`
+	RequiresApproval            bool           `json:"requiresApproval,omitempty"`
+	ContinuousDeployment        bool           `json:"continuousDeployment,omitempty"`
+	PullRequestPlanDeployments  bool           `json:"pullRequestPlanDeployments,omitempty"`
+	AutoDeployOnPathChangesOnly bool           `json:"autoDeployOnPathChangesOnly,omitempty"`
+	AutoDeployByCustomGlob      bool           `json:"autoDeployByCustomGlob,omitempty"`
+}
+
+type DeployRequest struct {
+	BlueprintId          string                `json:"blueprintId,omitempty"`
+	BlueprintRevision    string                `json:"blueprintRevision,omitempty"`
+	BlueprintRepository  string                `json:"blueprintRepository,omitempty"`
+	ConfigurationChanges *ConfigurationChanges `json:"configurationChanges,omitempty"`
```

Comment:
this one should be array of them

Reply:
so turns out it can be both, so i'll make it an array so were covered

Reply:
63d1a3c